startleft/api/controllers/iac/iac_create_otm_controller.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from http import HTTPStatus
+
+from fastapi import APIRouter, File, UploadFile, Form, Response
+
+from startleft.api.error_response import ErrorResponse
+from startleft.messages import messages
+from startleft.project.otm_project import OtmProject
+
+PREFIX = '/api/v1/startleft/iac'
+URL = ''
+RESPONSE_STATUS_CODE = HTTPStatus.CREATED
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix=PREFIX,
+    tags=["IaC"],
+    responses={
+        201: {"description": messages.OTM_SUCCESSFULLY_CREATED},
+        400: {"description": messages.BAD_REQUEST,
+              "model": ErrorResponse},
+        401: {"description": messages.UNAUTHORIZED_EXCEPTION,
+              "model": ErrorResponse},
+        403: {"description": messages.FORBIDDEN_OPERATION,
+              "model": ErrorResponse},
+        'default': {"description": messages.UNEXPECTED_API_ERROR,
+                    "model": ErrorResponse}
+    }
+)
+
+
+@router.post(URL, status_code=RESPONSE_STATUS_CODE)
+def iac(iac_file: UploadFile = File(..., description="File that contains the IaC File"),
+        iac_type: str = Form(..., description="Type of IaC File: CLOUDFORMATION, TERRAFORM, XML"),
+        id: str = Form(..., description="ID of the new project"),
+        name: str = Form(..., description="Name of the new project"),
```

Comment:
To be consistent, maybe we should avoid the word _project_, that is more related to the IriusRisk one. An alternative could be:
ID of the new threat model.
Name of the new threat model.